client/src/bg/background.js

```diff
@@ -4,10 +4,51 @@
 //     "sample_setting": "This is how you use Store.js to remember values"
 // });
 
+function generateUUID() {
+  let id = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
+    let r = Math.random()*16|0, v = c === 'x' ? r : (r&0x3|0x8);
+    return v.toString(16);
+  });
+  return id;
+}
```

Comment:
https://developer.chrome.com/apps/app_identity
We can use identity of chrome API. That can serve as a unique ID>